crates/server/src/authority/authority.rs

```diff
@@ -201,3 +198,103 @@ pub trait DnssecAuthority: Authority {
     /// Sign the zone for DNSSEC
     async fn secure_zone(&self) -> DnsSecResult<()>;
 }
+
+/// Result of a Lookup in the Catalog and Authority
+pub enum LookupResult<T, E = LookupError> {
+    /// A successful lookup result.
+    Ok(T),
+    /// An error was encountered during the lookup.
+    Err(E),
+    /// The authority did not answer the query and the next authority in the chain should
+    /// be consulted.  Do not use this for any other purpose, in particular, do not use it
+    /// to represent an empty lookup (use Ok(EmptyLookup) for that.)
+    Bypass,
```

Comment:
Is `Bypass` the best name for this? Is that what the standards use? I would also consider `Skip`, for example, given the documented semantics.

Reply:
I'm not aware of any references to this sort of behavior in the standards documents, but skip is fine with me.